app/models/offer.rb

```diff
@@ -1,3 +1,8 @@
 class Offer < ApplicationRecord
   belongs_to :request
+  has_many :reviews
+
+  validates :content, presence: true, length: { minimum: 25 }
+  validates :occurs_on, presence: true
+  validates :time, presence: true, numericality: { only_integer: true }
 end
```

Comment:
should we add a validation for status? can only be certain value (accepted/declined/pending)

Reply:
where is it that we put default status ? i.e. when we create an offer the status will be by default pending
I think it is somewhere in the model, or maybe migration?